gcloud-java-examples/src/main/java/com/google/gcloud/examples/nio/Stat.java

```diff
@@ -20,6 +20,9 @@
  * be logged in (using e.g. the gcloud auth command).
  *
  * <p>See the README for a command line to run this example.
+ *
```

Comment:
It does not feel right this duplicate.

Instead of a duplicate example did you consider replacing this one with something else simple?

If not, maybe we can add `contrib-nio-example` as a dependency and reference the code from here?